api/pom.xml

```diff
@@ -101,6 +101,13 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>webservices.rest-omod-common</artifactId>
 		</dependency>
+		
+		<dependency>
+ 	         <groupId>org.openmrs.module</groupId>
+ 	         <artifactId>reportingcompatibility-api</artifactId>
+ 	         <version>2.0.3</version>
```

Comment:
@moshonk I don't think we need that dependency, it is already available in the main pom